docs/dev/install.rst

```diff
@@ -267,6 +278,50 @@ you have to follow these steps:
 
 Once this is done, you should be able to trigger a new build on that project and it should succeed.
 
+
+Network connection failures
```

Comment:
I think this section goes too deep into pretty technical details that may not be required, making the section more confusing.

I've had similar problems and the solution that I applied was simpler: `inv docker.down` which deletes all the containers, networks and everything created by our own application.

IMO, telling people to modify `/etc/hosts` and debug using `route`, `docker network inspect` or similar is out of the scope and will only confuse users.

Reply:
It's a troubleshooting section, so I suppose that no one will read it unless they have "Network connection failures".

Great tip to remember to run `inv docker.down`, that should perhaps be the first simple step.

Reply:
Oh and I have removed the stuff about `/etc/hosts`, it's confusing and superstitious.

Reply:
Yeah, my point is that this is going too deep pretty quickly into a rabbit hole. Telling people to execute `docker network inspect community_readthedocs` which outputs a blob of JSON without telling them what to look at is more confusing than helpful.

Also, I don't think we want to deal with all of this together with the users. It seems a pretty edge case to have it here. As an example, I just executed `route` and I don't even have that command installed 🙃 

Besides, `docker network prune` will prune other Docker networks that are not created by Read the Docs, which could break other Docker setup the user may have. There are more, like telnet to the database without more instructions; but I don't think that writing _all of these instructions is the solution_, I think the solution is to not have this section 😄 

Summarizing, I'm +1 documenting the troubleshooting for the most common cases and guide the user towards the solution. On the other hand, I'm -1 on documenting edge cases like this one and go too deep on the debug. That will cause us more support requests to us than benefits to the users.

Reply:
Added `inv docker.down` as the first step.

Reply:
Yea, I'm on board with Manuel that this seems super deep and specific. I'm 99% sure we won't maintain this content as the commands change, and it will just become technical debt. I think documenting the simple case is worth doing, but the rest seems like nobody will ever need it. We can put that in an issue somewhere if you want it to be Googleable or something?

Reply:
Sure, will remove it :+1: Not sure if `route` will need to be maintained much :)

As a background, I have had several encounters where understanding that Docker subnets can have local issues becomes **super costly** in local debugging. In other encounters, I've had with this, the Docker subnet overlapped with a real subnet. That's why I added it, it was meant as a 101 to quickly debug such issues.

Reply:
Now it's removed.